cmd/pool/order.go

```diff
@@ -540,6 +547,8 @@ func printOrderDetails(client poolrpc.TraderClient, amt btcutil.Amount,
 		fmt.Printf("Self channel balance: %v\n", selfChanBalance)
 	}
 
+	fmt.Printf("Is public: %t\n", isPublic)
```

Comment:
nit: stray debug log statement.

Reply:
That was not supposed to be a debug log but to inform the user in the prompt when substituting an order.

In this way we can change the default value in the future and we make sure that users know what they are doing BEFORE submitting an order.

Reply:
Aaah, sorry, I didn't see that we're in the CLI. Never mind, then.